<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FastISPClass</class>
 <widget class="QMainWindow" name="FastISPClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>517</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>492</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FastISP</string>
  </property>
  <property name="windowIcon">
   <iconset resource="fastisp.qrc">
    <normaloff>:/FastISP/image/IC.png</normaloff>:/FastISP/image/IC.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>45</height>
       </size>
      </property>
      <property name="title">
       <string>联机下载时的程序文件</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_5" native="true">
         <property name="minimumSize">
          <size>
           <width>411</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>411</width>
           <height>16777215</height>
          </size>
         </property>
         <widget class="QToolButton" name="toolButton_path">
          <property name="geometry">
           <rect>
            <x>310</x>
            <y>4</y>
            <width>37</width>
            <height>18</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>37</width>
            <height>18</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>37</width>
            <height>18</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
         <widget class="QTextEdit" name="textEdit_path">
          <property name="geometry">
           <rect>
            <x>4</x>
            <y>1</y>
            <width>301</width>
            <height>24</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>301</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>301</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'宋体'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_4" native="true">
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QCheckBox" name="checkBox_reloadfile">
            <property name="text">
             <string>下载时重载文件</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item row="0" column="0">
        <widget class="QWidget" name="widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="widget_3" native="true">
            <layout class="QGridLayout" name="gridLayout">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item row="0" column="1">
              <widget class="QPlainTextEdit" name="plainTextEdit_output">
               <property name="readOnly">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QWidget" name="widget_2" native="true">
               <property name="minimumSize">
                <size>
                 <width>350</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>350</width>
                 <height>16777215</height>
                </size>
               </property>
               <layout class="QGridLayout" name="gridLayout_4">
                <property name="leftMargin">
                 <number>2</number>
                </property>
                <property name="topMargin">
                 <number>2</number>
                </property>
                <property name="rightMargin">
                 <number>2</number>
                </property>
                <item row="5" column="0" colspan="2">
                 <widget class="QWidget" name="widgetProcess" native="true">
                  <property name="minimumSize">
                   <size>
                    <width>332</width>
                    <height>26</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>332</width>
                    <height>26</height>
                   </size>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_4">
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="topMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <layout class="QHBoxLayout" name="ProcesshorizontalLayout">
                     <item>
                      <widget class="QLabel" name="label_4">
                       <property name="text">
                        <string>下载进度</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QProgressBar" name="progressBar">
                       <property name="value">
                        <number>24</number>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item row="0" column="0" colspan="2">
                 <widget class="QGroupBox" name="groupBox_3">
                  <property name="minimumSize">
                   <size>
                    <width>332</width>
                    <height>49</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>332</width>
                    <height>49</height>
                   </size>
                  </property>
                  <property name="title">
                   <string>串口配置</string>
                  </property>
                  <widget class="QComboBox" name="comboBox_comportname">
                   <property name="geometry">
                    <rect>
                     <x>46</x>
                     <y>18</y>
                     <width>101</width>
                     <height>22</height>
                    </rect>
                   </property>
                  </widget>
                  <widget class="QLabel" name="label">
                   <property name="geometry">
                    <rect>
                     <x>12</x>
                     <y>21</y>
                     <width>31</width>
                     <height>16</height>
                    </rect>
                   </property>
                   <property name="text">
                    <string>串口:</string>
                   </property>
                  </widget>
                  <widget class="QLabel" name="label_2">
                   <property name="geometry">
                    <rect>
                     <x>156</x>
                     <y>21</y>
                     <width>41</width>
                     <height>16</height>
                    </rect>
                   </property>
                   <property name="text">
                    <string>波特率:</string>
                   </property>
                  </widget>
                  <widget class="QComboBox" name="comboBox_baudrate">
                   <property name="geometry">
                    <rect>
                     <x>200</x>
                     <y>18</y>
                     <width>121</width>
                     <height>22</height>
                    </rect>
                   </property>
                  </widget>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QPushButton" name="pushButton_start">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>50</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>开始编程</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QPushButton" name="pushButton_refreshcom">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>50</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>搜索串口</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0" colspan="2">
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <item>
                   <widget class="QPushButton" name="pushButton_readicinf">
                    <property name="text">
                     <string>读取器件信息</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButton_clearic">
                    <property name="text">
                     <string>清除芯片</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButton_readflash">
                    <property name="text">
                     <string>读FLASH</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item row="3" column="0" colspan="2">
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string/>
                  </property>
                  <property name="pixmap">
                   <pixmap resource="fastisp.qrc">:/FastISP/image/def.png</pixmap>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0">
                 <widget class="QWidget" name="widget_7" native="true"/>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="fastisp.qrc"/>
 </resources>
 <connections/>
</ui>
